### PR TITLE
JENKINS-39498 Fix invalid data

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

File: src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/playback/GerritMissedEventsPlaybackManager.java [hunk 1]
```diff
@@ -155,6 +158,38 @@ protected synchronized Date getDateFromTimestamp() {
         return new Date();
     }
 
+    @Override
+    public void run() {
```
Comment (rsandell) [inline src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/playback/GerritMissedEventsPlaybackManager.java]: Probably better implemented as an [AsyncPeriodicWork](http://javadoc.jenkins.io/hudson/model/AsyncPeriodicWork.html) to save on thread count.

**Comment by scoheb** [comment]
@rsandell Sergii's tests were successful. Would you still like me to refactor to use AsyncPeriodicWork before we merge?

**Comment by rsandell** [comment]
@scoheb It's up to you. I can merge as is, and we can get that nit fixed sometime down the line.

**Comment by scoheb** [comment]
@rsandell refactoring complete. Take a look at your convenience.

**Comment by rsandell** [comment]
Thank you very much @scoheb for looking into this!

**Pull request merged**